### PR TITLE
Fix for #21693 - Rename install directory to 'glassfish5'

### LINKED ISSUE #21693: Rename install directory to 'glassfish5'
The GF ZIP uses glassfish4/ as root dir, this should now be 'glassfish5/'
#### Environment
NA
#### Affected Versions
[5.0]

### PR DESCRIPTION
**Pull request by arindam-bandyopadhyay**
Fix  #21693 - Rename install directory to 'glassfish5'

### CODE REVIEW

**Comment by arindam-bandyopadhyay** [comment]
@glassfishrobot Run CI tests please

File: main/appserver/packager/appserver-base/src/main/docs/README.txt [hunk 1]
```diff
@@ -1,4 +1,4 @@
-Thank you for downloading GlassFish Server Open Source Edition 4.1!
```
Comment (yaminikb) [inline main/appserver/packager/appserver-base/src/main/docs/README.txt]: Some of the other references to 4.1 in the same file which can be changed to 5.0

**Review by yaminikb — CHANGES_REQUESTED** [review]
Couple of comments:
1. You have missed changing the top-level README.md file.
2. I also found following files which have references to glassfish3, I think we should fix them as well:
./main/appserver/extras/embedded/glassfish-uber/build.xml
./main/appserver/tests/appserv-tests/config/emma.xml
./main/appserver/tests/appserv-tests/devtests/cdi/README_Embedded.txt
./main/appserver/tests/appserv-tests/devtests/cdi/smoke-tests/optional-unbundled-beans/foo.txt
./main/appserver/tests/appserv-tests/devtests/ejb/README
./main/appserver/tests/appserv-tests/devtests/transaction/ee/dblogs/base/readme
./main/appserver/tests/appserv-tests/devtests/transaction/ee/hudson_3.x.sh
./main/appserver/tests/appserv-tests/devtests/web/ha/cluster-tests/simple-failover-modified-attribute/src/main/java/com/acme/HttpClient.java
./main/appserver/tests/appserv-tests/devtests/web/ha/cluster-tests/simple-failover-modified-session/src/main/java/com/acme/HttpClient.java
./main/appserver/tests/appserv-tests/devtests/web/hudson-base.sh
./main/appserver/tests/appserv-tests/devtests/web/README_Embedded.txt
./main/appserver/tests/sse/basic/pom.xml
./main/appserver/tests/sse/videoplayer/pom.xml
./main/appserver/tests/sse/videoplayer/README.txt
./main/nucleus/core/logging/src/test/resources/com/sun/enterprise/server/logging/parser/uniform-server.log
./v2/appserv-tests/config/emma.xml
./v2/appserv-tests/devtests/hk2/README
./v2/appserv-tests/devtests/jdbc/connection-pool-perf/sample-console.log
./v2/appserv-tests/devtests/jms/README

Note we could leave out the v2/appserv-tests files.

3. One more file which needs changes is main/appserver/distributions/distributions.xml line #51

**Comment by romain-grecourt** [comment]
Make sure to squash when you accept the pull request.

File: main/appserver/packager/appserver-base/src/main/docs/README.txt (hunk 1 above)
Comment (arindam-bandyopadhyay) [inline main/appserver/packager/appserver-base/src/main/docs/README.txt]: This file has some more reference to 4.1. However those are mostly related to release notes . So I am not changing them

**Comment by arindam-bandyopadhyay** [comment]
@glassfishrobot Run CI tests please

**Review by yaminikb — APPROVED** [review]
Changes look good.

**Pull request merged**